ansible/pbTestScripts/buildJDK.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This seems very hardcoded to one particular variation.  If that's on purpose then that's OK but perhaps rename the file to reflect reality.  Else you can parameterize this script and pass arguments into it...

Reply:
Agree, although I've got so many other enhancements I would like to get on that I'd quite like to leave this as-is for now then adjust later if you're OK with that Martijn. I may even go for a random selection later. This is basically a simp!eat form of what's needed to run our builds using our environment scripts from openjdk-build. We're hoping to do something similar for running a basic operation fr. The test suites too

Reply:
OK :-)